### DISCUSSION TITLE
Hide blinding white background in some pages on webex.com

### DISCUSSION THREAD

**Discussion by DimitrisKalaboukas** [uBlockOrigin/uAssets#23703] [General] [answered]
If you open:

https://minedu-primary2.webex.com/mw3300/mywebex/cmr/cmr.do?siteurl=minedu-primary2&AT=meet&username=parasfoti&type=null

insert random name/email data, and click “Join by browser”, you will see one of them. Could anyone propose a rule to block the svg/xml based white background?

**Comment by krystian3w** [comment, marked as answer]
Have you tried using something like this:

```adb
##*:style(background-image: none ! important)
```

?

Initial * is bad in the long run - DOM Inspector is all red.

**Reply by DimitrisKalaboukas** [reply]
Yeah, it works fine, thanks a lot!
